wonderful_bing/wonderful_bing.py

```diff
@@ -37,8 +39,11 @@
 class WonderfulBing(object):
     def __init__(self, arguments):
         # Get all the information we need from this url, see issue#7
+        self.country = arguments['--country']
+        if not self.country:
+            self.country = 'en-US'
```

Comment:
Got it, the `arguments` passed by docopt will fill `--country` while the unittest not, that's why unit test failed.